### DISCUSSION TITLE
Tarjetas

### DISCUSSION THREAD

**Discussion by maumar2000** [BlackBoxVision/react-native-mercadopago-px#60] [General]
@JonatanSalas Hola Jonatan, que tal? Estoy haciendo la integración pero solo me muestra la opción de agregar una nueva tarjeta pero no me muestra las tarjetas que ya tengo guardadas. Como hago para que me muestre la tarjeta que ya tengo? Gracias!

**Comment by JonatanSalas** [comment]
Desgraciadamente el SDK sobre el cual basamos nuestra biblioteca no tiene soporte para persistencia de tarjetas. Podes leer un poco mas aca: 

https://github.com/mercadopago/px-android/issues/2322#issuecomment-691558160

**Reply by maumar2000** [reply]
@JonatanSalas Muchas gracias por tu rapida respuesta. Entonces debería usa esto https://github.com/BlackBoxVision/react-native-mercadopago-tokenize-checkout ?

**Reply by JonatanSalas** [reply]
De nada! Si, deberias ir por ahi, en esa lib nos falta un toque de documentacion de la integracion servidor, si necesitas un centro abrite issues o discussions que tiramos la info necesaria.
